src/components/shared/eventcard/eventcard.css.js

```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components';
+import { grid, space, color, typography, layout } from 'styled-system';
+import Text from 'components/shared/text';
+import Box from 'components/shared/box';
+
+// style-system default breakpoints
+// const breakpoints = ['40em', '52em', '64em']
+export const Article = styled.article`
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto auto 1fr;
+  grid-template-areas:
+    'date title'
+    'image image'
+    'rsvp rsvp';
+
+  @media screen and (min-width: 64em) {
+    grid-template-columns: auto 1fr;
+    grid-template-rows: auto auto 1fr;
+    grid-template-areas:
+      'date title'
+      'blank image'
+      'blank rsvp';
+  }
+
+  background-color: white;
+  box-sizing: border-box;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
If you're feeling up to it, stacking box shadows looks a lot better!

Reply:
Not sure I've ever done this before. Can you point me to something to read up on this?